js/app.js

```diff
@@ -363,16 +349,17 @@
 				});
 			}
 		},
+		addParticipantToCall: function(token, participant) {
+			return this.signaling.addParticipantToCall(token, participant);
```

Comment:
this should be were `createPublicVideoCall` is as well?
To make JS and php similar ( #349 )

It has nothing to do with signaling from my pov?